source/shared/net/mobile-wallet/remove-delegation-mw.rst

```diff
@@ -0,0 +1,31 @@
+.. _remove-delegation-mw:
+
+================================================
+Remove delegation on an account in Mobile Wallet
+================================================
+
+.. Note::
+
+    Currently delegation is only available on Testnet.
+
+.. Note::
+
+   All transfers and transactions cost a fee, including delegation transactions. The fee is based on the set NRG for that transaction and the current exchange rate.
+   The cost of transaction fees is stable in Euros, and therefore the price in CCD varies depending on the CCD to EUR exchange rate. The fee will always be deducted from the **Balance** of the account, so it is important to have some available CCDs to cover fees.
+   You can see the fee in the transaction log.
+
+#. Go to the **Accounts** screen.
+
+#. Tap on the balance area of the account for which you want to stop delegation or tap **More** (button picture).
+
+#. In the Hamburger menu (button picture) tap **Delegation**.
+
+#. You see your current delegation. Tap **Stop delegation**.
+
+#. Review the information in the transaction overview. When you are satisfied, tap **Submit delegation transaction**.
+
+#. Once the transaction is submitted you see a confirmation screen. Tap **Finish** to complete the action.
+
+.. Note::
+
+   Removing your stake in any pool results in a longer :ref:`cool-down period<glossary-cool-down-period>`. The delegation will be locked during this period and no changes can be made. The delegation stop is not effective until the cool-down period ends. During the cool-down period the staked amount continues earning rewards.
```

Comment:
It's actually only the amount that cannot be changed. I'm pretty sure that people can still change targets and restake status during the shut down cool-down